### ISSUE TITLE
metadata for timeline data

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/datahub#1320]
We have meta data for clinical attributes (e.g. name, type, priority, description, etc), but not fur timeline data. For example, we would like to visualize certain numerical timeline data (e.g. PSA and CEA) as line chart (as discussed in [this Slack private thread](https://cbioportal.slack.com/archives/C359NNB41/p1600095841095300)), but currently, we have to infer the data type (e.g. whether all data is numerical) and hardcode which property to use. 

It would be good to support meta data for the longitudinal dataset, but it could be more complex than the clinical data as we may have many properties for a data of one timepoint. One option is to define the meta data per property, ie. adding additional metadata lines in the timeline format (similar to clinical data)

**Comment by ritikakundra** [comment]
Yes agreed. Especially for genie, timeline only used normalized headers as display names.

**Comment by jjgao** [comment]
@ritikakundra @n1zea144 what would be the good next step?

**Comment by ritikakundra** [comment]
@jjgao we could start with first adding the attributes used in timeline files in our data dict with the metadata. Then the question is if we need to update the timeline file with those headers like the clinical files?

**Comment by jjgao** [comment]
@ritikakundra Good idea. Maybe we can start with the GENIE BPC cohort?
